sdk/purview/azure-purview-datamap/azure/purview/__init__.py

```diff
@@ -0,0 +1 @@
+__path__ = __import__("pkgutil").extend_path(__path__, __name__)  # type: ignore
```

Comment:
Please try to fix CI failure with https://github.com/Azure/azure-sdk-for-python/wiki/How-to-fix-CI

Reply:
updated